src/media/text_detector.py

```
@@ -0,0 +1,136 @@
+                return False, 0.0
+            
+            # Stage 2: OCR confidence check (slower validation)
+            ocr_data = pytesseract.image_to_data(
+                img,
+                output_type=pytesseract.Output.DICT,
+                config='--psm 6'  # Assume uniform text block
+            )
+            
+            # Filter confident words
+            confident_words = [
+                word for word, conf in zip(ocr_data['text'], ocr_data['conf'])
```

Ruff (0.14.4)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)